addons/ofxAssimpModelLoader/src/ofxAssimpModelLoader.cpp

```diff
@@ -388,20 +389,35 @@ void ofxAssimpModelLoader::loadGLResources(){
 				auto ogPath = texPathStr;
 				bool bHasEmbeddedTexture = false;
 
-				string modelFolder = ofFilePath::getEnclosingDirectory( file.path() );
-				string relTexPath = ofFilePath::getEnclosingDirectory(texPathStr,false);
-				string texFile = ofFilePath::getFileName(texPathStr);
-				string realPath = ofFilePath::join(ofFilePath::join(modelFolder, relTexPath), texFile);
+				// FIXME: Convert everything possible to fs. remove .string() in the end
+				auto modelFolder = ofFilePath::getEnclosingDirectory( file.path() );
+				auto relTexPath = ofFilePath::getEnclosingDirectory(texPathStr,false);
+				auto realPath = modelFolder / of::filesystem::path{ texPathStr };
+				
+				using std::cout;
+				using std::endl;
+				cout << "modelFolder : " << modelFolder << endl;
+				cout << "texPathStr : " << texPathStr << endl;
+				cout << "relTexPath : " << relTexPath.string() << endl;
+				cout << "realPath : " << realPath.string() << endl;
```

Comment:
probably want to remove this before merging

Reply:
Sure thing :)

Reply:
done!